### ISSUE TITLE
Welcome page does not properly preserve focus

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51976]
**Environment Details:**
VSCode Version : 1.23.1
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 2.4.3
Tools Used : Keyboard

**Repro Steps:**
1. Launch VS Code.
2. Navigate to Welcome screen.
3. Navigate to "Color Theme" and "Find and Run All Commands" button and press "Enter" to open popup.
4. Press Esc to close the popup and verify keyboard focus.

**Actual:**
Keyboard focus does not remain on the same button- "Color Theme" and "Find and Run All Commands"-(from which popup appears) once the popup is closed.

Focus order starts from top of the page. 

**Expected:**
Keyboard focus should remain on same button from which the popup appeared after closing it.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The user will again need to navigate through the entire screen and come back to same button or its next control to further perform actions.

**MAS Reference:**
MAS 2.4.3 - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

**Attachment for Reference:**
[A11y_VSCode_WelcomeScreen_Keyboard_FocusRetention.pptx](https://github.com/Microsoft/vscode/files/2105262/A11y_VSCode_WelcomeScreen_Keyboard_FocusRetention.pptx)

Does this issue occur when all extensions are disabled?: Yes
